option optimize_for = LITE_RUNTIME;

message DrawVipDailyWelfareRequest
{
 required uint32 id = 1;
}
